buildrunner/steprunner/tasks/build.py

```diff
@@ -54,21 +54,25 @@ def __init__(
 
         buildrunner_config = BuildRunnerConfig.get_instance()
 
-        if self.cache_from:
+        #  Pull the cache_from images only for single platform builds
+        if self.cache_from and isinstance(self.cache_from, list) and not self.platforms:
             for cache_from_image in self.cache_from:
-                try:
-                    self._docker_client.pull(cache_from_image, platform=self.platform)
-                    # If the pull is successful, add the image to be cleaned up at the end of the script
-                    self.step_runner.build_runner.generated_images.append(
-                        cache_from_image
-                    )
-                    self.step_runner.log.write(
-                        f"Using cache_from image: {cache_from_image}\n"
-                    )
-                except Exception:  # pylint: disable=broad-except
-                    self.step_runner.log.write(
-                        f"WARNING: Unable to pull the cache_from image: {cache_from_image}\n"
-                    )
+                if isinstance(cache_from_image, str):
```

Comment:
Oh, this is in the single platform code, gotcha.

Reply:
I could change it to support multiplatform and it should be beneficial for our build nodes.

Reply:
I'll create a new ticket to track this, so we can test it too.